docs/_data/prague-2021-schedule.yaml

```diff
@@ -0,0 +1,78 @@
+writing_day:
```

Comment:
The CEST is aligned weirdly under the starting time, not sure if it's actually needed inline...